nss-day-cohort-41/tabloid-fullstack-elvis_spotters
# Js edit a tag

# Description
Implemented modal for editing tags. Currently only admins are able to edit.
Fixes #16 
## Type of change
# Testing Instructions
- Pull down PR branch
- Modify SQL express connection string and FirebaseProjectId in appsettings local file if needed
- Launch API and App
- Navigate to tag management 
- Click on edit button on tag you wish to modify
- Modal should appear with the tag name to be edited
- Make changes and click save, should be re-directed back to tag list with updated tags
- Edit modal won't allow user to submit an empty field
- Test above steps with non-admin user, should receive an unauthorized message in console
# Checklist:
